src/types/Verification.ts

```diff
@@ -31,8 +31,16 @@ export class VerificationSession extends String {
  * The message result of [[requestPresentation]] which should be sent to the [[Claimer]] and used in [[buildPresentation]].
  */
 export class PresentationRequest extends String {
-  // @ts-ignore
-  private thisIsOnlyHereToPreventClassMixes: int
+  public getRequestedProperties(): string[] {
```

Comment:
A test is missing for this. Do you want to do that or should I?

Reply:
To be more precise: Lines 32-39 and 63.

Reply:
i will add a test for that